Analyses/Applications - McAfee ePolicy Orchestrator - Configuration - Windows

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<BES xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="BES.xsd">
+	<Analysis>
+		<Title>Applications - McAfee ePolicy Orchestrator - Configuration - Windows</Title>
+		<Description>Provides configuration information for McAfee ePO.</Description>
```

Comment:
Can you change the description to:
<Description><![CDATA[<P>Provides configuration information for McAfee ePO.</P>
<P>For general information or to report issues with C3 Protect content please visit GitHub here: <A href="https://github.com/strawgate/C3-Protect">https://github.com/strawgate/C3-Protect</A></P>]]></Description>